### ISSUE TITLE
MiKTeX portable 21.8: XeLaTeX won't compile (a problem with extractbb?)

### ISSUE THREAD

**Issue by cj-lai** [MiKTeX/miktex#920]
- [ ] I have read and acknowledged this HOWTO article:  
  [Submitting a MiKTeX problem report](https://miktex.org/howto/bug-report)

Hi, after reinstalling the latest portable miktex 21.8 on a win 10 device, I was compiling a very simple tex file with texworks as below:
```
\documentclass[12pt]{article}
\begin{document}
Test
\end{document}
```
While it works perfectly via pdfLaTeX, I've been seeing the following error messages as I compile via XeLaTeX:

```
(D:\Dropbox\Portable\MikTeX\texmfs\install\tex/latex/l3backend\l3backend-xetex.
def ()
! Argument of \__sys_tmp:w has an extra }.
<inserted text> 
                \par 
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
```
Keep pressing enter for all the ``missing'' arguments leads to the following messages below:
```
Runaway argument?
'extractbb' is not recognized as an internal or external command, ope\ETC.
! Paragraph ended before \__sys_tmp:w was complete.
<to be read again> 
                   \par 
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
! Missing number, treated as zero.
<to be read again> 
                   \scan_stop: 
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
! Missing = inserted for \ifnum.
<to be read again> 
                   \tex_par:D 
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
! Missing number, treated as zero.
<to be read again> 
                   \tex_par:D 
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 

! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
! You can't use `\numexpr' in horizontal mode.
\int_compare:nNnTF ...nt_eval:w #1#2\__int_eval:w 
                                                  #3\__int_eval_end: \__prg_...
l.80 ...c__kernel_sys_dvipdfmx_version_int { 0 } }
                                                  
? 
```
It ends up outputting a document with an undesired < at the beginning (see [MWE.pdf](https://github.com/MiKTeX/miktex/files/7130424/MWE.pdf)).
I suspect that this is the same problem as in 
https://stackoverflow.com/questions/65986999/latex-runtime-error-with-xelatex-compiler
However, the solutions there won't work for me.

Let me also attach the report text and relevant logs.
[miktex-report.txt](https://github.com/MiKTeX/miktex/files/7130429/miktex-report.txt)
[xelatex.log](https://github.com/MiKTeX/miktex/files/7130433/xelatex.log)
[MWE.log](https://github.com/MiKTeX/miktex/files/7130418/MWE.log)

I'd appreciate it for any help.

**Comment by edocevoli** [comment]
Thank you. I was not able to reproduce the problem with you test case.

**Comment by se-pri** [comment]
The same mistake happened to me today after I updated my packages in the MiKTeX Console. I use MiKTeX 21.12.10 on Windows 10, using XeLaTeX as my standard compiler. The same errors appeared after I updated my packages today which I haven't done for some months. My PDF file looks like yours. Attached you find my MiKTeX report.
Is there any solution to this problem?
[miktex-report.txt](https://github.com/MiKTeX/miktex/files/7822118/miktex-report.txt)
